undirect.go

```diff
@@ -125,7 +125,7 @@ func (g Undirect) Weight(x, y Node) (w float64, ok bool) {
 	re := g.G.Edge(y, x)
 
 	var f, r float64
-	if wg, ok := g.G.(Weighter); ok {
+	if wg, wOk := g.G.(Weighter); wOk {
```

Comment:
Now I remember that I saw this during the review but then somehow forgot to comment. Sorry.